README.rst

```diff
@@ -81,6 +88,18 @@ This MFE has 2 flags of its own:
 * ``SUPPORT_URL_LEARNER_RECORDS`` -- A link to a help/support center for learners who run into problems whilst trying to share their records
 * ``USE_LR_MFE`` -- A toggle that when on, uses the MFE to host shared records instead of the the old UI inside of credentials
 
+Verifiable Credentials
+......................
+
+An optional feature. It is behind a feature flag.
```

Comment:
Please add the way it can be configured using the `MFE_ENVIRONMENT_EXTRA` variable
https://github.com/openedx/configuration/blob/open-release/nutmeg.master/playbooks/roles/mfe_deployer/tasks/main.yml#L16

Reply:
No changes are required, Operators would find a way to put this feature flags in nodeenv as there's only one way for configuring MFEs env during deployment